tests/bluetooth/bsim_bt/bsim_test_eatt/src/main_reconfigure.c

```diff
@@ -10,12 +10,16 @@
 
 #include <zephyr/bluetooth/gatt.h>
 
+#define NEW_MTU 100
+
 CREATE_FLAG(flag_reconfigured);
 
 void att_mtu_updated(struct bt_conn *conn, uint16_t tx, uint16_t rx)
 {
 	printk("MTU Updated: tx %d, rx %d\n", tx, rx);
-	SET_FLAG(flag_reconfigured);
+	if (rx == NEW_MTU || tx == NEW_MTU) {
+		SET_FLAG(flag_reconfigured);
+	}
```

Comment:
Again test case execution change, have in separate PR and be merged before switch to refactor PR.

Reply:
See PR #48859